examples/webview_custom_protocol.rs

```diff
@@ -62,9 +64,11 @@ impl WebViewDelegate for WebViewInstance {
 }
 
 struct AppWindow {
+    #[cfg(feature = "webview")]
```

Comment:
I'm not a big fan of doing it this way but think it might worth the convenience of a simple `cargo test`. I don't think you can specify that a given example is for a given feature flag or os target (in the case of the ios example) right?

Reply:
May this works?

```toml
[[example]]
name = "webview_custom_protocol"
required-features = ["webview"]
```

Reply:
Neat! That works!